### DISCUSSION TITLE
librispeech训练效果不好

### DISCUSSION THREAD

**Discussion by zdyupup** [PaddlePaddle/PaddleSpeech#760] [Q&A]
环境：paddlepaddle2.0.2+cuda10.0
训练用的语音数据集：train-clean-100.tar.gz+train-clean-360.tar.gz，共460小时

30+epoch之后train_loss在50-150之间，infer.sh输出的wer值为0.34
我想请教一下，一般英文语音识别数据集选取多少？训练有效的train_loss和wer值标准是多少？

**Comment by zh794390558** [comment]
可以参看 examples/librispeech
